doc/release-notes.md

```diff
@@ -263,6 +263,13 @@ sanity check. Since 0.12, these are no longer stored. When loading a
 0.12 wallet into an older version, it will automatically rescan to avoid
 failed checks.
 
+Wallet: Pruning
+---------------
+
+With 0.12 it is possible to use basic wallet functionality in prunded mode.
```

Comment:
I think we should remove `basic`. Expect of the import and rescan features, you can use everything. `Basic` may imply that only a smaller subset of functions are available.